library/core/src/intrinsics.rs

```diff
@@ -2550,3 +2550,13 @@ pub const unsafe fn write_bytes<T>(dst: *mut T, val: u8, count: usize) {
         write_bytes(dst, val, count)
     }
 }
+
+#[cfg(not(bootstrap))]
+extern "rust-intrinsic" {
+    /// The offset of the `Some(_)` value of an `Option<T>`. Used internally for
```

Comment:
Please re-add the FIXME from the hacky libstd impl here, so we remember to nuke this intrinsic in the future